package.json

```diff
@@ -29,6 +28,8 @@
   },
   "homepage": "https://grammy.dev",
   "dependencies": {
-    "@go-task/go-npm": "0.1.18"
+    "@go-task/go-npm": "0.1.18",
```

Comment:
Remove dependency, we'll not use that when this PR got merged.

Reply:
Ohh this might have been left by mistake fixing it ASAP